codalib/util.py

```diff
@@ -294,7 +294,7 @@ def updateQueue(destinationRoot, queueDict, debug=False):
         print uploadXMLText
     try:
         response, content = doWebRequest(url, "PUT", data=uploadXMLText)
-    except:
+    except (urllib2.URLError, urllib2.HTTPError):
```

Comment:
Since `urllib2.HTTPError` is a subclass of `urllib2.URLError`, it should be the same to use:
    `except urllib2.URLError:`